tests/integration_tests/functional/test_cpu_features.py

```diff
@@ -197,6 +197,12 @@ def test_brand_string(test_microvm_with_api):
         assert False
 
 
+# The X2APIC MSRs cannot to be loaded and stored on VMX transitions. A VMX transition fails
```

Comment:
```diff
-cannot to be loaded
+cannot be loaded
```

Reply:
This is a direct quote from https://courses.cs.washington.edu/courses/cse451/24wi/documentation/x2apic.pdf the mistake is also present there.

Reply:
I think we should make it clear that it is a quote, otherwise it's going to be very tempting to fix the next time we touch this.